core/templates/pages/classroom-page/classroom-page.controller.ts

```diff
@@ -54,7 +54,12 @@ angular.module('oppia').directive('classroomPage', [
             ctrl.bannerImageFileUrl = UrlInterpolationService.getStaticImageUrl(
               '/splash/books.svg');
 
-            PageTitleService.setPageTitle(ctrl.classroomName + ' - Oppia');
+            ctrl.classroomDisplayName = (
+              ctrl.classroomName.charAt(0).toUpperCase() +
```

Comment:
Can we use a filter to capitalize on the texts? (I think we have one, can you please check?)

Reply:
Done.